style.css

```diff
@@ -0,0 +1,989 @@
+  text-decoration: underline;
+}
+
+.news-source {
+  display: flex;
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
+
+.deals-source {
+  display: flex;
+}
+
+.price {
+  font-size: 13px;
+  color: rgb(163, 162, 162);
+}
+
+.author,
+.price {
+  margin-right: 10px;
+}
+
+.author:hover {
+  color: black;
+}
+
+.img-1 {
+  background-image: url("./images/1.jpg");
+}
+
+.img-2 {
+  background-image: url("./images/3.jpg");
+}
+
+.img-3 {
+  background-image: url("./images/4.jpg");
+}
+
+.img-4 {
+  background-image: url("./images/5.png");
+}
+
+.img-5 {
+  background-image: url("./images/6.jpg");
+}
+
+.img-6 {
+  background-image: url("./images/7.jpeg");
+}
+
+.img-7 {
+  background-image: url("./images/8.jpeg");
+}
+
+.img-8 {
+  background-image: url("./images/111.jpg");
+}
+
+.funding-sec,
+.groups {
+  max-width: 1176px;
+  margin: 20px auto;
+  padding: 0 20px;
+}
+
+.f-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.f-title ul {
+  display: flex;
+  list-style-type: none;
+}
+
+.f-title li:hover {
+  color: black;
+}
+
+.left-border {
+  border-left: 1px solid #f42;
+}
+
+.deal-1 {
+  background-image: url("./images/a-female-office-worker-at-the-table-holding-papers-with-diagrams-796x531.jpg");
+}
+
+.deal-2 {
+  background-image: url("./images/Apple-Amazon-Facebook-Google-hed-796x375.jpg");
+}
+
+.deal-3 {
+  background-image: url("./images/bitcoin_today-header_bitcoin_today.jpg");
+}
+
+.deal-4 {
+  background-image: url("./images/BMW-ix3-unveiling-car-future-bavarian-motor-ev-electric-796x448.png");
+}
+
+.deal-5 {
+  background-image: url("./images/Facebook-Discover-796x419.jpg");
+}
+
+.deal-6 {
+  background-image: url("./images/222.png");
+}
+
+.deal-7 {
+  background-image: url("./images/Legend-of-Korra-796x417.jpg");
+}
+
+.deal-8 {
+  background-image: url("./images/product_38387_product_shots1.jpg");
+}
+
+.deal-holder {
+  width: 100%;
+  padding-top: 74.5%;
+  background-size: cover;
+  background-position: center;
+}
+
+footer {
+  text-align: center;
+}
+
+.footer-2 {
+  padding: 39px;
+  background-color: black;
+  color: #aaa;
+  font-size: 12px;
+}
+
+.footer-2 img,
+.footer-2 a {
+  height: 9px;
+}
+
+.icons > * {
+  color: white;
+  background-color: #333;
+  border-radius: 50%;
+  padding: 12px;
+  border: 1px solid #333;
+  margin: 0 8px;
+}
+
+.footer-1 > * {
+  padding: 15px 0;
+}
+
+.after-image > * {
+  color: #f42;
+  font-size: 14px;
+}
+
+.group-title-stuff > * {
+  color: white;
+  padding-right: 14px;
+}
+
+.mid-funding > * {
+  color: #555;
+  padding: 5px 10px;
+  font-size: 14px;
+}
+
+.icons > *:hover {
+  background-color: #131313;
+}
+
+.icons {
+  justify-content: center;
+  display: flex;
+}
+
+.footer-list {
+  display: flex;
+  justify-content: center;
+}
+
+.footer-1 {
+  background-color: #131313;
+  padding: 20px 0 10px 0;
+}
+
+.funding-info {
+  margin: 20px 0;
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(4, 1fr);
+  grid-column-gap: 25px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:grid;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.d-grid{
  display: grid;
}
```
**Html file**

```
<div class="d-grid">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_